docs/source/user-guide/sql/window_functions.md

```diff
@@ -0,0 +1,204 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either expressioness or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Window Functions
+
+Window functions calculate a value for a set of rows within a result set.
```

Comment:
We can probably copy some/all of the description from the postgres docs rather than having to reinvent it: https://www.postgresql.org/docs/current/tutorial-window.html

Reply:
Thanks for the feedback! I stitched together a brief description of window functions from the postgres docs and added a sub-section describing the syntax.